scripts/smoketest.py

```diff
@@ -45,6 +46,9 @@ def run(command, **kwargs):
 
 sample_id = str(uuid.uuid4())
 bundle_dir = "data-bundle-examples/10X_v2/pbmc8k"
+with open(os.path.join(bundle_dir, "large_file"), "wb") as fh:
+    fh.write(os.urandom((1024 * 1024 * 64) + 1))
```

Comment:
Are you explicitly disconnecting this script from the rest of the DSS code base?

Reply:
No, I don't intend to do that. Do you have suggestions for what I should use here?

Reply:
Let me make this threshold a constant first.

Reply:
https://github.com/HumanCellAtlas/data-store/pull/447

Reply:
Addressed, thanks. Please re-review